packages/dd-trace/src/appsec/remote_config/manager.js

```diff
@@ -299,4 +319,22 @@ function parseConfigPath (configPath) {
   }
 }
 
+function supportsAckCallback (handler) {
+  if (kSupportsAckCallback in handler) return handler[kSupportsAckCallback]
```

Comment:
you're caching the result/allowing overriding right ?

Reply:
Yes, checking for the rest-operator unfortunately requires us to read the function source code, so to now have to do that more than once, I cache the result.

Not sure what you mean by the last part of your question ("allowing overriding")?

Reply:
well it's not really meant for it, but if i get a hold of the symbol, i can force the RC manager to consider a handler supporting ack or not, might be useful one day. Resolving the thread...